phpassignment-2/functions.php

```diff
@@ -0,0 +1,95 @@
+<?php
+    require ("vendor/autoload.php");
+
+    /**
+     * Help to get data and images from the json file.
+     * 
+     */
+    class CallApi {
+        // Stores all the data elements.
+        public $arr; 
+
+        /**
+         * @var $client creates an object of GuzzleHTTP.
+         * @var $host stores the hostname. 
+         * @var $response stores response after making GET request.
+         * @var $body  
+         * @var $arr stores the response in array format.
+         */
+        function __construct() {
+            $client = new \GuzzleHttp\Client();                                    
+            $host = 'https://ir-dev-d9.innoraft-sites.com/jsonapi/node/services';  
+            $response = $client->request('GET', $host);
+            $body = $response->getBody();
+            $arr = json_decode($body,TRUE);
+            $this->arr = $arr;
```

Comment:
can we make a separate function to fetch data. 
Also we can define $client as class variable so we don't have to create Client object multiple time.